projects/plugins/jetpack/_inc/client/discussion/subscriptions.jsx

```diff
@@ -134,6 +139,17 @@ class SubscriptionsComponent extends React.Component {
 									'jetpack'
 								) }
 							/>
+							<ToggleControl
+								checked={ isSubscriptionsActive && this.props.getOptionValue( 'sm_enabled' ) }
+								disabled={
+									! isSubscriptionsActive ||
+									unavailableInOfflineMode ||
+									this.props.isSavingAnyOption( [ 'subscriptions' ] )
+								}
+								toggling={ this.props.isSavingAnyOption( [ 'sm_enabled' ] ) }
+								onChange={ this.handleSubscribeModalToggleChange }
+								label={ __( 'Enable the popup Subscribe Modal', 'jetpack' ) }
```

Comment:
Copy feels a bit clunky, but I don't have alternatives to suggest now. Non blocking merge.

Reply:
I would agree. The capitalization was what jumped at me. Maybe this could at least be:

```suggestion
								label={ __( 'Enable the popup subscription modal', 'jetpack' ) }
```

Reply:
Erick is probably following the pattern of the two previous settings.

<img width="555" alt="Screenshot 2023-07-05 at 18 12 52" src="https://github.com/Automattic/jetpack/assets/104869/8f8168a8-68b2-44ea-8617-88fc4352d7a1">